src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -2744,7 +2764,14 @@ private bool AnyEventEnabled()
             return false;
         }
 
-        private bool IsDisposed { get { return m_provider == null || m_provider.m_disposed; } }
+        private bool IsDisposed 
+        {
+#if FEATURE_MANAGED_ETW
+            get { return m_provider == null || m_provider.m_disposed; }
+#else
+            get { return true; }
```

Comment:
IsDisposed should return false when ETW is not present.   (IsDisposed means that the EventSource is 'off' (broken))